src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -321,6 +336,23 @@ func (gen *PythonStartupScriptGenerator) buildPtvsdCommandForModule(moduleAndArg
 	return cdcmd + pycmd
 }
 
+func (gen *PythonStartupScriptGenerator) buildDebugPyCommandForModule(moduleAndArgs string, appDir string) string {
+	waitarg := ""
+	if gen.DebugWait {
+		waitarg = "--wait-for-client"
+	}
+
+	cdcmd := ""
+	if appDir != "" {
+		cdcmd = fmt.Sprintf("cd %s && ", appDir)
```

Comment:
Will this handle paths with spaces?

Reply:
I didn't see any shell escaping in any of the other examples. probably out of scope for this PR, but worth asking the maintainers.